### PR TITLE
Make IRS report legend colors customizable

### LINKED ISSUE #646: RVL-734 Change Colour Gradient Range for Dashboards
New color gradients should be:

- Grey: 0%
- Red: 0 to 70%
- Yellow: 70 to 80%
- Light Green: 80 to 85%
- Green: >85%

Current values: https://github.com/onaio/reveal-frontend/blob/master/src/configs/settings.ts#L848

This change should only affect Namibia.

More info: https://smartregister.atlassian.net/browse/RVL-734

### PR DESCRIPTION
**Pull request by kelvin-muchiri**
Closes #646

### CODE REVIEW

**Comment by moshthepitt** [comment]
@kelvin-muchiri it would be mice to have a way, in tests, to show that:

- Namibia colors are rendered correctly when using Namibia settings
- default colors are rendered correctly when not using Namibia settings

**Comment by kelvin-muchiri** [comment]
> * Please fix broken tests
> * Why is the App component affected at all? (might be an issue with your env vars)

There is one test failing. @p-netm confirmed it was failing before. Though he said he'll look into it

**Comment by moshthepitt** [comment]
> There is one test failing. @p-netm confirmed it was failing before. Though he said he'll look into it

@kelvin-muchiri @p-netm as far as I know we have no failing tests on the master branch.

**Comment by cKellyDesign** [comment]
@kelvin-muchiri was this branched off of `master`? Not sure how it would have existing failing tests.

File: src/configs/settings.ts [hunk 1]
```diff
@@ -870,6 +870,45 @@ export const indicatorThresholdsIRS: IndicatorThresholds = {
...
+    name: 'Red',
+    orEquals: true,
+    value: 0.7,
+  },
+  YELLOW_THRESHOLD: {
+    color: '#FFDC00',
+    name: 'Yellow',
+    value: 0.8,
+  },
+};
```
Comment (cKellyDesign) [inline src/configs/settings.ts]: Can we pass the `name` strings in from the `lang.ts` file so we are sure they are translated?

File: src/App/tests/__snapshots__/App.test.tsx.snap [hunk 2]
```diff
@@ -25,7 +25,8 @@ exports[`App renders App correctly 1`] = `
         "listen": [Function],
         "location": Object {
           "hash": "",
-          "pathname": "/",
+          "key": "nmani6",
+          "pathname": "/login",
```
Comment (cKellyDesign) [inline src/App/tests/__snapshots__/App.test.tsx.snap]: Looks like the App redirecting to `/login` when it mounts in the test?

File: src/App/tests/__snapshots__/App.test.tsx.snap [hunk 3]
```diff
@@ -846,370 +851,161 @@ exports[`App renders App correctly 1`] = `
...
+                          path="/login"
+                          render={[Function]}
                         >
-                          <PrivateRoute
-                            authenticated={false}
-                            component={[Function]}
-                            computedMatch={
+                          <OauthLogin
+                            ProviderLinksComponent={[Function]}
+                            history={
```
Comment (cKellyDesign) [inline src/App/tests/__snapshots__/App.test.tsx.snap]: Seems like these shouldn't be changing

**Comment by kelvin-muchiri** [comment]
> @kelvin-muchiri it would be mice to have a way, in tests, to show that:
> 
> * Namibia colors are rendered correctly when using Namibia settings
> * default colors are rendered correctly when not using Namibia settings

Yea..sure..

**Comment by moshthepitt** [comment]
> Great thanks @kelvin-muchiri - just one more step to add these to the i18n formatted en.json file.

Good catch @cKellyDesign.  What can we do to ensure that something like this is caught in a failing test in the future?

File: src/configs/settings.ts (hunk 1 above)
Comment (cKellyDesign) [reply src/configs/settings.ts]: I've added these to a ticket requesting TH translations for new display strings.

**Comment by cKellyDesign** [comment]
> What can we do to ensure that something like this is caught in a failing test in the future?

@moshthepitt is that covered by this ticket? https://github.com/onaio/reveal-frontend/issues/639

**Comment by moshthepitt** [comment]
> @moshthepitt is that covered by this ticket? #639

Yes, my bad!

**Pull request merged**